src/MLJDecisionTreeInterface.jl

```diff
@@ -39,9 +45,13 @@ MMI.@mlj_model mutable struct DecisionTreeClassifier <: MMI.Probabilistic
     post_prune::Bool             = false
     merge_purity_threshold::Float64 = 1.0::(_ ≤ 1)
     display_depth::Int           = 5::(_ ≥ 1)
+    feature_importance::Symbol = :impurity::(_ ∈ (:none, :impurity, :split)) #:permutation))
```

Comment:
We don't need to add the `:none` option here, the other two options are okay.
Also, MLJ has just recently exposed a standard API for feature importance. We hope to use this in more complicated oprations such as Recursive feature elimination, etc. 
Following these steps to adhere to the API specs.
1. First you need to declare that a model supports feature importance computation. This is done by defining
```julia
import MLJModelInterface as MMI
MMI.reports_feature_importances(::Type{<:DecisionTreeClassifier}) = true`
```
2. Then you need extend the `MMI.feature_importances(model, fitresult, report)` method for the `DecisionTreeClassfier` model type, by defining something along these lines
```julia
function MMI.feature_importances(model::DecisionTreeClassifer, fitresult, report)
    # do computation that would return feature importances
end
```
For more information on the requirements that your function should adhere to see [here](https://github.com/JuliaAI/MLJModelInterface.jl/blob/dev/src/model_api.jl#L169-L183)